proto/regen/ecocredit/v1/query.proto

```diff
@@ -89,15 +95,6 @@ service Query {
   }
 }
 
-// QueryParamsRequest is the Query/Params request type.
-message QueryParamsRequest {}
-
-// QueryParamsResponse is the Query/Params response type.
-message QueryParamsResponse {
-  // params defines the parameters of the ecocredit module.
-  Params params = 1;
-}
-
```

Comment:
Why are these being removed? Maybe this was a mistake while switching between branches?